README.md

```diff
@@ -126,6 +126,34 @@ params={{
       }}
 ```
 
+##### `customizeWidget`
+
+- All colors are specified using predefined [color names](https://www.w3schools.com/colors/colors_names.asp), or RGB, HEX, HSL, RGBA, HSLA values.
+- All `width` and `height` values can be any of the [standard css lenght unit types](https://www.w3schools.com/cssref/css_units.asp).
+
+| Prop / Param                  | Default value  | Description |
```

Comment:
I would add a new column with `Type` of the field, that would be `color` or `dimension`.